### ISSUE TITLE
sending multiple long sysex msg causes audio dropouts

### ISSUE THREAD

**Issue by mariobuoninfante** [pure-data/pure-data#914]
MacOS High Sierra 10.13.1 - Pd 0.50.2

I noticed that sending 2 long sysex (>150 bytes) in fast succession (see screenshot below) causes audio dropouts.

<img width="496" alt="Screen Shot 2020-03-25 at 18 16 27" src="https://user-images.githubusercontent.com/20879405/77571162-ca021180-6ec4-11ea-9c38-3233e453b300.png">

**Comment by mariobuoninfante** [comment]
this causes dropouts as well
<img width="250" alt="Screen Shot 2020-03-25 at 18 18 51" src="https://user-images.githubusercontent.com/20879405/77571343-1a796f00-6ec5-11ea-9477-4f59dad3ac1c.png">

**Comment by mariobuoninfante** [comment]
forgot to mention that the delay option in the audio settings is set to 15 msec, when set to 20 the issue disappears

**Comment by mariobuoninfante** [comment]
could this be related in any way to https://github.com/pure-data/pure-data/issues/912?

**Comment by Spacechild1** [comment]
Does this still happen in Pd 0.51?

**Comment by mariobuoninfante** [comment]
yes, just tried on MacOS High Sierra 10.13.6 running Pd 0.51-0, same results
